test/Parse/line-directive.swift

```diff
@@ -30,8 +30,8 @@ public struct S {
 // expected-error@+5{{operators must have one or two arguments}}
 // expected-error@+4{{member operator '/()' must have at least one argument of type 'S'}}
 // expected-error@+3{{expected '{' in body of function declaration}}
-// expected-error@+2 2 {{consecutive declarations on a line must be separated by ';}}
-// expected-error@+1 2 {{expected a macro identifier}}
+// expected-error@+2 {{consecutive declarations on a line must be separated by ';}}
+// expected-error@+1 {{expected a macro identifier}}
 / ###line 25 "line-directive.swift"
```

Comment:
This is improved because previously `#` _not_ followed by an identifier was `true` for `isStartOfSwiftDecl()`. Now it's `false` so the expression parsing recovery skips the second `#`.

Reply:
Makes sense, thank you!